### PR TITLE
Add PSMDB 4.2 jobs and minor PSMDB fixes

### PR DESCRIPTION
**Pull request by tplavcic**
Added 4.2 template + trunk and param job.
Removed from template jobs some logic which is not used any more.
Jobs are already updated in jenkins.

### CODE REVIEW

**Comment by tplavcic** [comment]
@hors 
I'm getting some strange issue with the trunk job, it just stalls in preparation stage: https://psmdb.cd.percona.com/job/percona-server-for-mongodb-4.2-trunk/2/console
maybe you know what's the problem

**Comment by puneet0191** [comment]
@tplavcic not completely sure but it seems like an issue with a process that is started in background and didn't end, or stuck

**Comment by tplavcic** [comment]
@puneet0191 
yes, seems it gets stuck in some `apt` step:
```
ubuntu    5371  0.0  0.0  13312  3100 ?        S    14:02   0:00 /bin/bash /tmp/jenkins936076626878483959.sh
root      5623  0.0  0.0  63972  4136 ?        S    14:02   0:00 sudo bash -x psmdb_builder.sh --builddir=/mnt/jenkins/workspace/percona-server-for-mongodb-4.2-trunk/test --repo=https://github.com/percona/percona-server-mongodb.git --branch=v4.2 --install_deps=1
root      5624  0.0  0.0  13840  3708 ?        S    14:02   0:00 bash -x psmdb_builder.sh --builddir=/mnt/jenkins/workspace/percona-server-for-mongodb-4.2-trunk/test --repo=https://github.com/percona/percona-server-mongodb.git --branch=v4.2 --install_deps=1
root      6184  0.0  0.1 110180 60588 ?        S    14:02   0:02 apt-get -y install python python-dev valgrind scons liblz4-dev devscripts debhelper debconf libpcap-dev libbz2-dev libsnappy-dev pkg-config zlib1g-dev libzlcore-dev dh-systemd libsasl2-dev gcc g++ cmake curl libssl-dev libcurl4-openssl-dev
root      8471  0.0  0.0  21296  5068 pts/0    Ss+  14:03   0:00 /usr/bin/dpkg --status-fd 41 --configure --pending
root      8593  0.0  0.0  65412 19004 pts/0    S+   14:03   0:00 /usr/bin/perl -w /usr/share/debconf/frontend /var/lib/dpkg/info/libssl1.1:amd64.postinst configure 1.1.0g-2ubuntu4.3
root      8605  0.0  0.0   4628  1692 pts/0    S+   14:03   0:00 /bin/sh /var/lib/dpkg/info/libssl1.1:amd64.postinst configure 1.1.0g-2ubuntu4.3
```
looks related to the message in the console output, maybe it's waiting for user input for some reason.

File: psmdb/percona-server-for-mongodb-4.2-template.yml [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+        killall -9 mongos || true
+        killall -9 mongod || true
+        pkill -f killer.sh || true
+    - shell: |
+        #!/bin/bash
+
+        set -x
+
+        if [ -f /usr/bin/apt-get ]; then
+            sudo apt-get -y install wget
```
Comment (hors) [inline psmdb/percona-server-for-mongodb-4.2-template.yml]: sudo  DEBIAN_FRONTEND=noninteractive apt-get -y install wget

**Pull request merged**